ACCESS-OM2-GMD-Paper-Figs/Fig12-ZonalTempSaltBias.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>This could be replaced with:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">import sys
sys.path.insert(0, "/g/data/tm70/rb5533/cosima-recipes/ACCESS-OM2-GMD-Paper-Figs")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>It could be, but I am not sure what the best way... </p>